datagrepper/app.py

```diff
@@ -96,6 +96,13 @@ def inject_variable():
     return extras
 
 
+@app.after_request
+def add_cors(response):
```

Comment:
You want to restrict this to GET routes for now (not sure if datagrepper takes any other though HTTP methods though).

Reply:
Yeah, this is only sufficient if the browser never wants to make a "pre-flight request" using OPTIONS. Basically as long as your clients are only making GET requests with no unusual headers...

https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#Examples_of_access_control_scenarios section "Simple requests"

Otherwise, you would need to answer the OPTIONS requests. I guess *that* complexity is what the flask-cors package is supposed to handle for you.